tests/test_tw_vote.py

```diff
@@ -0,0 +1,114 @@
+"""
+Tests for triggerable withdrawals voting.
+"""
+
+from typing import Dict, Tuple, List, NamedTuple
+from scripts.tw_vote import create_tw_vote
+from brownie import interface, convert, web3, ZERO_ADDRESS
+from utils.test.tx_tracing_helpers import *
+# from utils.config import contracts, LDO_HOLDER_ADDRESS_FOR_TESTS
+from utils.config import (
+    VALIDATORS_EXIT_BUS_ORACLE_IMPL,
+    WITHDRAWAL_VAULT_IMPL,
+    LIDO_LOCATOR_IMPL,
+    LDO_HOLDER_ADDRESS_FOR_TESTS,
+    contracts,
```

Comment:
Please do not use utils.config addresses in tests - re-define variables from scratch in this file

Reply:
We can leave it as is for testnet deployment but I will ask you to do it for mainnet deployment without exceptions. See the screenshot for the rationale:
![image](https://github.com/user-attachments/assets/e73e1301-f55d-4f66-8365-ef4ace9d2a9e)